executor/src/witgen/processor.rs

```diff
@@ -62,7 +62,7 @@ pub struct OuterQuery<'a, 'b, T: FieldElement> {
 impl<'a, 'b, T: FieldElement> OuterQuery<'a, 'b, T> {
     pub fn try_new(
         caller_rows: &'b RowPair<'b, 'a, T>,
-        connection: Connection<'a, T>,
+        bus_receive: &'a BusReceive<T>,
```

Comment:
Stuck here. This needs the LHS to evaluate it. Why doesn't the caller do this again?